index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Alice Cieslik . Persönliche Website</title>
+  </head>
+  <body>
+    <header>
+      <p><strong>Alice Cieslik . Persönliche Website</strong> </p>
+
+      <p>Ich baue eine <strong>Website</strong>.</p>
+    </header>
+
+    <nav>
+      <strong>Navigation</strong> 
+      <ul>
+        <li>Home</li>
+        <li>Über mich</li>
+        <li>Dienstleistungen</li>
+        <li>Blog</li>
+        <li>Kontakt</li>
+      </ul>
+    </nav>
+
+    <main>
+      <h1>Hier steht der Hauptinhalt</h1>
```

Comment:
```suggestion
      <h1>Hier steht der Hauptinhalt</h1>
```
This one is not necessary as your heading above already describes the purpose of your website and the `h1` element is usually part of the `header`. :)